seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1084,7 +1084,7 @@
         "/* head */ 'use strict'; /* solution */ var myStr = '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>'; /* tail */ (function() { return \"myStr = \" + myStr; })();"
       ],
       "tests": [
-        "assert(!/\\\\/g.test(code) && myStr === '<a href=\"http://www.example.com\" target=\"_blank\">Link</a>', 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
+        "assert(!/\\\\/g.test(code) && myStr.match('\\\\s*<a href=\"http://www.example.com\"\\\\s*target=\"_blank\">\\\\s*Link\\\\s*</a>\\\\s*'), 'message: Remove all the <code>backslashes</code> (<code>\\</code>)');",
```

Comment:
Do you think we should be precautions and allow spaces around for example the `=` characters as well?

The following string will not pass the tests at this time. Note the space after `target=`
```js
var myStr ='<a href="http://www.example.com" target= "_blank">Link</a>';
```